core/lib.deno_core.d.ts

```diff
@@ -115,5 +115,23 @@ declare namespace Deno {
     function setMacrotaskCallback(
       cb: () => bool,
     ): void;
+
+    /**
+     * Set a callback that will be called when a promise without a .catch
+     * handler is rejected.
+     */
+    function setPromiseRejectCallback(
+      cb: (type: number, promise: Promise, reason: any) => void,
+    );
+
+    /**
+     * Set a callback that will be called when an exception isn't caught
+     * by any try/catch handlers. Currently only invoked when the callback
+     * to setPromiseRejectCallback() throws an exception but that is expected
+     * to change in the future.
+     */
+    function setUncaughtExceptionCallback(
+      cb: (err: any) => void,
+    );
```

Comment:
I realize now the function prototypes are white lies because I made them return the old handlers. I'll update before merging.